src/Main.elm

```diff
@@ -182,13 +197,29 @@ updateLoading msg model =
                             ( modelFraLoadingState { state | cv = Just cv }, Cmd.none )
 
                         Err error ->
-                            ( Failure error, Cmd.none )
+                            ( Failure error
+                            , logFeilmelding error "Opprett CV"
+                            )
 
                 _ ->
                     ( model, Cmd.none )
 
         RegistreringsProgresjonHentet result ->
-            ( model, Cmd.none )
+            case result of
+                Ok value ->
+                    ( model, Cmd.none )
+
+                Err error ->
+                    ( Failure error
+                    , logFeilmelding error "Hent registreringsprogresjon"
+                    )
+
+
+logFeilmelding : Http.Error -> String -> Cmd Msg
+logFeilmelding error message =
+    Feilmelding.feilmelding "message" error
```

Comment:
Her har du hermetegn rundt `message` sånn at feltet `operasjon` i feilmeldinger  alltid blir strengen `"message"` i stedet for det som sendes med til `logFeilmelding` funksjonen, så det må du fikse. I tillegg kan du godt endre navn på argumentet `message` til `operasjon`, ettersom det er det det heter i feilmelding 😊